Insulation_and_boundaries.ipynb

```diff
@@ -44,6 +44,27 @@
     "from cooltools.insulation import calculate_insulation_score, find_boundaries"
```

Comment:
<p>What do others think, should we call average ChIP-seq profile a pileup, or is it confusing, when we use the same word for Hi-C?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>I think using the same word is good, but maybe we could clarify as 1D pileup or 2D pileup</p>

Reply:
+1  , i'm okay with 1D pileups vs 2d pileups - they have exactly same math, right?

Reply:
<p>Clarified it in the annotation, ok!</p>

Reply:
yeah, it's confusing - all this pileup - stackup - etc terminology. Do people outside of our circle even use terms "pileup" and/or "stackup"?

Reply:
<p>No. My examiner (computational biologist!) was confused by "pileup". People only know "pileup" for pileup of reads: http://www.htslib.org/doc/samtools-mpileup.html</p><p> </p>